src/components/core/tests/PivotTable.spec.tsx

```diff
@@ -294,6 +295,45 @@ describe("PivotTable", () => {
             expect(createProvider).toHaveBeenNthCalledWith(2, false);
         });
     });
+
+    describe("isTableHidden", () => {
+        const wrapper = renderComponent();
+        const table = wrapper.find(PivotTableInner);
+        const tableInstance = table.instance() as any;
+
+        it("should return true if columnDefs are empty", () => {
+            expect(tableInstance.isTableHidden()).toEqual(true);
+        });
+
+        it("should return false if columnDefs are not empty", () => {
+            tableInstance.setState({ columnDefs: [{}] });
+            expect(tableInstance.isTableHidden()).toEqual(false);
+        });
+    });
+
+    describe("onModelUpdated", () => {
+        const wrapper = renderComponent();
+        const table = wrapper.find(PivotTableInner);
+        const tableInstance = table.instance() as any;
+        const updateStickyRow = jest.spyOn(tableInstance, "updateStickyRow");
```

Comment:
this is `updateStickyRow, updateStickyHeadersPosition` mocked instances is shared between tests. I would say is not safe approach ...

Reply:
Yeah i should call mockRestore at end of test right?